Evergreen/Manifests/AmazonCorretto.json

```diff
@@ -0,0 +1,49 @@
+{
+	"Name": "Amazon Corretto",
+	"Source": "https://aws.amazon.com/corretto/",
+	"Get": {
+		"Update": {
+			"Uri": ""
+		},
+		"Download": {
+			"JDK": {
+				"8": {
+					"x64msi" :"https://corretto.aws/downloads/latest/amazon-corretto-8-x64-windows-jdk.msi",
+					"x64zip" :"https://corretto.aws/downloads/latest/amazon-corretto-8-x64-windows-jdk.zip",
```

Comment:
Are the zip files used for deployments? I don't know enough about the JDK deployment to answer this one myself. If used, leave them in.

Reply:
You're probably right - I don't use them myself and they cause 6 additional invoke-webrequests. I've removed them to speed up the function